modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -958,7 +963,7 @@ public String toApiName(String name) {
             return "DefaultApi";
         }
         name = sanitizeName(name);
-        return camelize(name) + "Api";
+        return camelize(name) + apiNameSuffix;
```

Comment:
Note: This will impact all spring code gen, not just cloud.

Reply:
Yes but the default is "Api", correct? So nothing should happen for the other ones. The tests said it's okay :P

I think it make sense also for the other spring generator to add a name suffix if you want. Nothing special to spring-cloud